### DISCUSSION TITLE
custom theme or adjust the default theme

### DISCUSSION THREAD

**Discussion by dirkeinecke** [slint-ui/slint#7187] [Q&A]
Hi!

Is there a guide on how to create a custom theme or adjust the default theme used? I'm working on a macOS app and would like to make the interface look more aligned with the standard macOS appearance.

Dirk

**Comment by ogoffart** [comment]
On macOS, the default "cupertino" Slint style should be fairly aligned with the standard macOS appearance. Of course it's not perfect and there are room for improvements. (We even have an issue to track such differences: https://github.com/slint-ui/slint/issues/5844 )
If you feel you can help to improve the cupertino style to make it more aligned with macOS, we'd be happy to accept your contributions.

There isn't really a guide, but the code of the cupertino style is in https://github.com/slint-ui/slint/tree/master/internal/compiler/widgets/cupertino

**Comment by microhobby** [comment]
I kind of have the same question, but I'm trying to change the color accents of the FluentPalette. Is there some way to do it?

**Comment by FaintentDev** [comment]
no u cant change it. use exported globals and a struct for different theme styles, and u kinda have to create your own components instead of std-widgets
